src/component/page/deltakerliste-veileder/FilterMenyVeiledertype.tsx

```diff
@@ -0,0 +1,58 @@
+import { Checkbox, CheckboxGroup, Panel } from '@navikt/ds-react'
+import React from 'react'
+
+import { useTiltaksoversiktSokStore } from '../../../store/tiltaksoversikt-sok-store'
+import { klikkFilterMeny, loggKlikk } from '../../../utils/amplitude-utils'
+import styles from './FilterMenyDeltakerliste.module.scss'
+
+interface Props {
+	erMedveilederMap: Map<boolean, number>
+	className?: string
+}
+
+export const FilterMenyVeiledertype = (props: Props): React.ReactElement => {
+	const {
+		erMedveilederFilter,
+		leggTilMedveileder,
+		fjernFraMedveileder,
+	} = useTiltaksoversiktSokStore()
+
+	const VeiledertypeCheckbox = ({ erMedveileder } : { erMedveileder: boolean }) => {
+		const antallDeltakere = props.erMedveilederMap.get(erMedveileder) ?? 0
+
+		return  (
+			<Checkbox
+				className={styles.checkbox}
+				name="filter-veiledertype"
+				checked={erMedveilederFilter.includes(erMedveileder)}
+				onChange={(e) => {
+					if (e.target.checked) {
```

Comment:
Jeg tror jeg får en warning her: ```
react-dom.development.js:86 Warning: Invalid value for prop `value` on <input> tag. Either remove it from the element, or pass a string or number value to keep it in the DOM. For details, see https://reactjs.org/link/attribute-behavior 
    at input
    at div
    at http://127.0.0.1:3001/deltakeroversikt/node_modules/.vite/deps/chunk-R6L3YDMU.js?v=5c5c1a78:12342:49
    at VeiledertypeCheckbox (http://127.0.0.1:3001/deltakeroversikt/src/component/page/deltakerliste-veileder/FilterMenyVeiledertype.tsx?t=1678697814701:29:5)
```
Er det sånn at noe ikke liker at jeg bruker boolean for filteret..?

Reply:
Jeg tror du har en liten bug bare, du sender inn funksjonen `medveilederTekst` i seg selv som value, isteden for å kalle funksjonene for å sette riktig value 🥲 

Jeg lurer i tillegg på om det går an å skrive seg vekk fra å bruke propen `checked` da vi får my warnings på det.

Reply:
Nice, tusen takk!! :D 

Hmm, checked er ikke brukt her: https://aksel.nav.no/komponenter/core/checkbox, så jeg burde jo egentlig klare å skrive meg bort fra å bruke den 🤔

Reply:
Jeg tror kanskje det å skrive oss bort fra checked må gjøres i en egen oppgave, for det er allerede ganske mye endringer her, og for meg er det iallefall ikke helt rett frem å gjøre det.